ext/ldap/ldap.stub.php

```diff
@@ -4,281 +4,281 @@
 #endif
 
 #if (LDAP_API_VERSION > 2000) || defined(HAVE_ORALDAP)
-/** @param resource $link_identifier */
-function ldap_rename($link_identifier, string $dn, string $newrdn, string $newparent, bool $deleteoldrdn, array $servercontrols = []): bool {}
+/** @param resource $ldap */
+function ldap_rename($ldap, string $dn, string $new_rdn, string $new_parent, bool $delete_old_rdn, array $controls = []): bool {}
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @return resource|false
  */
-function ldap_rename_ext($link_identifier, string $dn, string $newrdn, string $newparent, bool $deleteoldrdn, array $servercontrols = []) {}
+function ldap_rename_ext($ldap, string $dn, string $new_rdn, string $new_parent, bool $delete_old_rdn, array $controls = []) {}
 
 
 /**
- * @param resource $link_identifier
+ * @param resource $ldap
  * @param array|string|int $retval
  */
-function ldap_get_option($link_identifier, int $option, &$retval = null): bool {}
+function ldap_get_option($ldap, int $option, &$retval = null): bool {}
 
 /**
- * @param resource|null $link_identifier
- * @param array|string|int|bool $newval
+ * @param resource|null $ldap
+ * @param array|string|int|bool $value
  */
-function ldap_set_option($link_identifier, int $option, $newval): bool {}
+function ldap_set_option($ldap, int $option, $value): bool {}
 
 /**
- * @param resource $link_identifier
- * @param resource $result_identifier
+ * @param resource $ldap
+ * @param resource $result
  */
-function ldap_count_references($link_identifier, $result_identifier): int {}
+function ldap_count_references($ldap, $result): int {}
 
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param resource $result
  * @return resource|false
  */
-function ldap_first_reference($link, $result) {}
+function ldap_first_reference($ldap, $result) {}
 
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param resource $entry
  * @return resource|false
  */
-function ldap_next_reference($link, $entry) {}
+function ldap_next_reference($ldap, $entry) {}
 
 #ifdef HAVE_LDAP_PARSE_REFERENCE
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param resource $entry
  * @param array $referrals
  */
-function ldap_parse_reference($link, $entry, &$referrals): bool {}
+function ldap_parse_reference($ldap, $entry, &$referrals): bool {}
 #endif
 
 #ifdef HAVE_LDAP_PARSE_RESULT
 /**
- * @param resource $link
+ * @param resource $ldap
  * @param resource $result
  * @param int $errcode
  * @param string $matcheddn
  * @param string $errmsg
  * @param array $referrals
- * @param array $serverctrls
+ * @param array $controls
  */
-function ldap_parse_result($link, $result, &$errcode, &$matcheddn = null, &$errmsg = null, &$referrals = null, &$serverctrls = null): bool {}
+function ldap_parse_result($ldap, $result, &$errcode, &$matcheddn = null, &$errmsg = null, &$referrals = null, &$controls = null): bool {}
```

Comment:
```suggestion
function ldap_parse_result($ldap, $result, &$error_code, &$matched_dn = null, &$error_message = null, &$referrals = null, &$controls = null): bool {}
```

`$errno` could also be used, but so far `$error_code` is what we preferred in another PR.